src/ompl/util/RandomNumbers.h

```diff
@@ -139,6 +146,30 @@ namespace ompl
             return localSeed_;
         }
 
+        /** \brief Uniform random sampling of a unit-length vector. I.e., the surface of an n-ball */
+        void uniformNormalVector(unsigned int n, double value[]);
```

Comment:
Could we use *value here? And maybe clarify in the comment that the memory is expected to exist